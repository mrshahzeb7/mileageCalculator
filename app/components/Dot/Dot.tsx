import React from 'react';
import {StyleProp, StyleSheet, View, ViewStyle} from 'react-native';
import {responsiveFontSize, responsiveWidth} from '../../utils';
import {colors} from '../../theme';

interface Props {
  style?: StyleProp<ViewStyle>;
}
const dotSize = responsiveFontSize(2);
export const Dot: React.FC<Props> = ({style}) => {
  return <View style={[styles.dot, style]} />;
};

// styles
const styles = StyleSheet.create({
  dot: {
    height: dotSize,
    width: dotSize,
    borderRadius: dotSize / 2,
    backgroundColor: colors.primaryButton,
    marginLeft: responsiveWidth(5.5),
  },
});
