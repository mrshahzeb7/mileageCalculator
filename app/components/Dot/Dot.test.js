import 'react-native';
import React from 'react';
import {Dot} from './Dot';
import renderer from 'react-test-renderer';

describe('Dot', () => {
  it(' Dot renders correctly', () => {
    const tree = renderer.create(<Dot />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
