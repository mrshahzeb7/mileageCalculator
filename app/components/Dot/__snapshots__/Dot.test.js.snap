// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dot  Dot renders correctly 1`] = `
<View
  style={
    [
      {
        "backgroundColor": "#93b2fd",
        "borderRadius": 15.297966458904849,
        "height": 30.595932917809698,
        "marginLeft": 41.25,
        "width": 30.595932917809698,
      },
      undefined,
    ]
  }
/>
`;
